content/en/docs/overview/_index.md

```diff
@@ -19,47 +17,83 @@ aliases:
+
+### Eliminates configuration mismanagement
+Development teams risk configuration inconsistencies when promoting workloads between environments that run on different technology stacks. For example, if you're running a testing environment with Docker Compose and a production environment on Kubernetes, keeping your Wwrkloads' configuration in sync can be challenging as each platform comes with its own set of APIs, semantics, syntax and configuration constructs. With Score, developers describe their workloads once with `score.yaml` and any required platform-specific configuration can be automatically generated via a Score Implementation CLI (e.g. score-compose or score-helm). This significantly reduces the risk of configuration mismatch between environments.
+
+### Enables separation of concerns
+Score enables a clear separation of concerns between developer-owned Workload related configuration and platform-owned infrastructure related configuration: Developers describe what their Wwrkload requires to run as part of `score.yaml` and if the requirements are honored by the platform, the workload runs as intended.
 
 ## What Score is not
 
-Score exclusively takes care of translating the Workload requirements specified in `score.yaml` into a platform-specific format. This means:
+Score exclusively takes care of translating the workload requirements specified in `score.yaml` into a platform-specific format. This means:
+
+- Score is **not a configuration management tool** for environments. It isn't recommended to store configuration values or secrets in `score.yaml`. Instead, it is possible to declare items such as configuration maps or secrets and vaults as a workload dependency in your Score file. These can then be resolved through the target runtime.
 
-- Score is **not a configuration management tool** for environments. It isn't recommended to store configuration values or secrets in `score.yaml`. Instead, it is possible to declare items such as configuration maps or secrets and vaults as a Workload dependency in your Score file. These can then be resolved through the target runtime.
 - Score is **not a resource and environment management system**, such as Terraform, or an Internal Developer Platform built around a Platform Orchestrator like Humanitec. It won’t spin up or purge physical resources or services defined in `score.yaml`.
-- Score is **not a deployment tool**. It doesn't support you with deploying or promoting Workloads across environments.
 
-Score will seamlessly integrate with the tech and tools you already have in place for configuration, resource, environment and deployment management. It does not intend to replace any part of your tech stack, but instead allows developers to feed in required Workload configuration parameters through the Score Implementation CLI in an automated and accessible way.
+- Score is **not a deployment tool**. It doesn't support you with deploying or promoting wrkloads across environments.
```

Comment:
While the first part is definitely correct, I've always found the second statement somewhat confusing. Score does support me in deploying across environments by being an input to these deployments, which are performed by _some other mechanism_.

So while this is strictly off-topic for the rearrangement of our content, can we make an adjustment while we're at it? Here's a take:

- Score is **not a deployment tool**. It does not perform deployments or the promoting of workloads across environments. Rather, it will be an input to these processes.

Reply:
@TobiasBabin Totally agree - I updated the copy to match your suggestion.